Makefile

```diff
@@ -13,6 +13,7 @@ GOPROXY     ?= https://proxy.golang.org
 # systems gsed won't be installed, so will use sed as expected.
 SED     ?= $(shell which gsed 2>/dev/null || which sed)
 GIT     ?= $(shell which git)
+CMP     ?= $(shell which cmp)
```

Comment:
why not use what linting does just by using git, this way will reduce the dependancies.

Reply:
I didn't saw it, I'll use git instead!